libraries/standard/mqtt/src/mqtt.c

```diff
@@ -41,6 +41,11 @@
     #define MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT    ( 5U )
 #endif
 
+/**
+ * @brief A return code indicating an error from the transport interface.
+ */
+#define TRANSPORT_RETURNED_ERROR    ( -1 )
```

Comment:
`TRANSPORT_ERROR` would probably be a better name?

Reply:
That's what I had originally, but then I changed it for some reason. Changing it back.